### PR TITLE
Add My Fun Countdown Site

### PR DESCRIPTION
**Pull request by sampoder**
Inspired by SpaceX Launch added this fun little countdown site

### CODE REVIEW

**Comment by sampoder** [comment]
Preview: https://summer-site.sampoder.now.sh/countdown

File: vercel.json [hunk 1]
```diff
@@ -6,5 +6,8 @@
       "source": "/api/prereg",
       "headers": [{ "key": "Access-Control-Allow-Origin", "value": "*" }]
     }
+  ],
+  "redirects": [
```
Comment (lachlanjc) [inline vercel.json]: You’ll want to use `rewrites` here so the routes don’t redirect
```suggestion
  "rewrites": [
```

File: pages/countdown.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
```
Comment (lachlanjc) [inline pages/countdown.html]: This file can be deleted—pages in this directory can’t be raw HTML

File: public/countdown.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css"
+    />
+    <title>Countdown to Summer of Making</title>
+    <style>
+      body {
+        margin: 0px;
+        font-family: Phantom Sans;
```
Comment (lachlanjc) [inline public/countdown.html]: Recommend doing `Phantom Sans, system-ui, sans-serif` so the text displays even if the font fails to load
```suggestion
        font-family: Phantom Sans, system-ui, sans-serif;
```

File: public/countdown.html [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+      }
+      button:hover {
+          -webkit-animation: grow 1s;
+          animation: grow 1s;
+      }
+      .grid-container {
+        display: grid;
+        grid-template-columns: 30% 70%;
```
Comment (lachlanjc) [inline public/countdown.html]:
```suggestion
        grid-template-columns: 3fr 7fr;
```

File: public/countdown.html [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+        height: 100vh;
```
Comment (lachlanjc) [inline public/countdown.html]:
```suggestion
        min-height: 100vh;
```

File: public/countdown.html [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+      }
+      .grid-item {
+        background-color: black;
+        font-size: 30px;
+        height: 100vh;
```
Comment (lachlanjc) [inline public/countdown.html]:
```suggestion
        height: 100%;
```

File: public/countdown.html [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+        font-family: "Phantom Sans";
+        src: url("https://assets.hackclub.com/fonts/Phantom_Sans_0.7/Regular.woff")
+            format("woff"),
+          url("https://assets.hackclub.com/fonts/Phantom_Sans_0.7/Regular.woff2")
+            format("woff2");
+        font-weight: normal;
+        font-style: normal;
+        font-display: swap;
+      }
+      @font-face {
```
Comment (lachlanjc) [inline public/countdown.html:52]: Doesn’t look like you’re using the italic version—can delete this

File: public/countdown.html [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+        line-height: inherit;
+        -webkit-text-decoration: none;
+        text-decoration: none;
+        font-size: inherit;
+        padding-left: 16px;
+        padding-right: 16px;
+        padding-top: 8px;
+        padding-bottom: 8px;
+        color: var(--theme-ui-colors-white, #ffffff);
+        background-color: var(--theme-ui-colors-primary, #ec3750);
```
Comment (lachlanjc) [inline public/countdown.html]:
```suggestion
```

File: public/countdown.html [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+        text-align: center;
+        line-height: inherit;
+        -webkit-text-decoration: none;
+        text-decoration: none;
+        font-size: inherit;
+        padding-left: 16px;
+        padding-right: 16px;
+        padding-top: 8px;
+        padding-bottom: 8px;
+        color: var(--theme-ui-colors-white, #ffffff);
```
Comment (lachlanjc) [inline public/countdown.html]:
```suggestion
```

**Review by lachlanjc — COMMENTED** [review]
Can’t wait to see this! A few code suggestions.

**Comment by sampoder** [comment]
Thank you @lachlanjc! The rocket is taking off 🚀

**Pull request merged**